src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -9,11 +9,144 @@
 
 package common
 
-import "fmt"
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/container/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"golang.org/x/sync/errgroup"
+)
 
 const FeedbackMsg = `We'd love to hear about your experience with redpanda:
 https://vectorized.io/feedback`
 
 func DeprecationMessage(altCmd string) string {
 	return fmt.Sprintf("use '%s' instead.", altCmd)
 }
+
+// Try to read the config from the default expected locations, or from the
+// specific path passed with --config. If --config wasn't passed, and the config
+// wasn't found, return the default configuration.
+func FindConfigFile(
+	mgr config.Manager, configFile *string,
+) func() (*config.Config, error) {
+	var conf *config.Config
+	var err error
+	return func() (*config.Config, error) {
+		if conf != nil {
+			return conf, nil
+		}
+		conf, err = mgr.ReadOrFind(*configFile)
+		if err != nil {
+			log.Debug(err)
```

Comment:
It might be useful to let the user know if ReadOrFind succeeded, can log at info, @0x5d what do you think?

Reply:
AFAIK, if there is an error cobra return the error and show a message